File: URI Online Judge/C-Exercises/URI 1044.c
#include <stdio.h>

int main()
{
    int a, b, resul;
    scanf("%i %i", &a, &b);
    if (a > b)
    {
        resul = a % b;
    }
    else
    {
        resul = b % a;
    }
    if (resul == 0)
    {
        printf("Sao Multiplos\n");
    }
    else
    {
        printf("Nao sao Multiplos\n");
    }

    return 0;
}

File: URI Online Judge/Python/URI 1044.py
x=input().split()
a,b=x
a=int(a)
b=int(b)
if b>a:
    resul=b%a
else:
    resul=a%b
    
if resul==0:
    print("Sao Multiplos")
else:
    print("Nao sao Multiplos")